lib/carrierwave/uploader/content_type_blacklist.rb

```diff
@@ -14,7 +14,7 @@ module ContentTypeBlacklist
       #
       # === Returns
       #
-      # [NilClass, Array[String,Regexp]] a blacklist of content types which are not allowed to be uploaded
+      # [NilClass, String, Regexp, Array[String,Regexp]] a blacklist of content types which are not allowed to be uploaded
```

Comment:
For consistency, can you add a space in `Array[String,Regexp]` so that it becomes `Array[String, Regexp]`.